### PR TITLE
ODIM 5194 Added omitempty in json tag

### PR DESCRIPTION
**Pull request by irappa-pattar**

### CODE REVIEW

**Review by jeevan-kamkar — APPROVED** [review]

**Review by skhan-anjum — APPROVED** [review]

File: lib-dmtf/model/ComputerSystem.go [hunk 1]
```diff
@@ -416,35 +416,35 @@ URIs:
...
-	SerialNumber    string        `json:"SerialNumber,omitempty"`
-	SKU             string        `json:"SKU,omitempty"`
-	Status          Status        `json:"Status,omitempty"`
+	Oid             string         `json:"@odata.id"`
+	Ocontext        string         `json:"@odata.context,omitempty"`
+	Oetag           string         `json:"@odata.etag,omitempty"`
+	Otype           string         `json:"@odata.type,omitempty"`
+	Description     string         `json:"description,omitempty"`
+	ID              string         `json:"Id,omitempty"`
+	Name            string         `json:"Name,omitempty"`
```
Comment (amar-shalgar) [inline lib-dmtf/model/ComputerSystem.go]: Please remove omitempty from "@odata.id", "@odata.type", "Id", "Name". These are required fields

**Review by amar-shalgar — CHANGES_REQUESTED** [review]
Please address the changes

File: lib-dmtf/model/ComputerSystem.go (hunk 1 above)
Comment (irappa-pattar) [reply lib-dmtf/model/ComputerSystem.go]: addressed

**Review by amar-shalgar — APPROVED** [review]
LGTM

**Review by bharath-b-hpe — APPROVED** [review]

**Review by bharath-b-hpe — APPROVED** [review]

**Pull request merged**